.github/workflows/service_test_s3.yml

```diff
@@ -178,9 +178,9 @@ jobs:
       - name: Test
         shell: bash
         working-directory: core
-        run: cargo nextest run s3
+        run: cargo nextest run behavior
         env:
-          OPENDAL_S3_TEST: on
+          OPENDAL_TEST: s3
```

Comment:
@Xuanwo It seems we should align Java's behavior test with this option change?

If so, may you create an issue and ping me >_<

Reply:
DONE

Reply:
Tracked at https://github.com/apache/incubator-opendal/issues/3267